start.py

```diff
@@ -144,6 +152,27 @@ def set_openpype_global_environments() -> None:
     if "QT_AUTO_SCREEN_SCALE_FACTOR" not in os.environ:
         os.environ["QT_AUTO_SCREEN_SCALE_FACTOR"] = "1"
 
+    # --- Set environment variables to vendorized binaries(FFmpeg and OIIO) ---
+    # TODO add validations of existing binaries
+    # Prepare path to bin vendor directory
+    bin_vendor_dir = os.path.join(BUILD_ROOT, "vendor", "bin")
+    # Prepare platform name
+    platform_name = platform.system().lower()
+
+    # FFmpeg path
+    ffmpeg_path_parts = [bin_vendor_dir, "ffmpeg", platform_name]
+    # Windows version has executables in `bin` directory
+    if platform_name == "windows":
+        ffmpeg_path_parts.append("bin")
+
+    ffmpeg_path = os.path.join(*ffmpeg_path_parts)
+
+    os.environ["OPENPYPE_FFMPEG_PATH"] = ffmpeg_path
+
+    # OIIO path
+    oiio_path = os.path.join(bin_vendor_dir, "oiio", platform_name)
+    os.environ["OPENPYPE_OIIO_PATH"] = oiio_path
+
```

Comment:
question is: should we set special environment variables for this? We have two other options - one is more complex but more flexible and the other one is more simple but maybe more universal:
1) create api function to return path to **oiio** or **ffmpeg** as we had in previous version I believe - this will allow more logic behind setting path to this and won't break if we somehow clean environment, rename variable or what not.
2) add those `bins` directly to `PATH`. This is simpler, allows user to override it with his own versions - for example when testing new features  - but it is more prone to error.

Also it would be cool for future additions to just handle `bin` directories there no matter if it is **oiio**, **ffmpeg** or **foobar** in future releases without changing this code?

Reply:
Option 2 is very dangerous but I 100% agree option 1 (api functions).

> should we set special environment variables for this
Environments were used because I didn't have any ability to access root path of build (code) from openpype module.